script_mteb_french/preload_models.py

```diff
@@ -0,0 +1,38 @@
+"""This script is used to download all models, for them to be used even without internet"""
+
+
+SENTENCE_TRANSORMERS_MODELS = [
```

Comment:
in file get_model_specs.py S**ENTENCE_TRANSFORMERS_MODELS** is without S : **SENTENCE_TRANSFORMER_MODELS**
standardize it